include/configs/odroid-g12-common.h

```diff
@@ -126,6 +127,40 @@
         "loadaddr=1080000\0"\
         "outputmode=1080p60hz\0" \
         "hdmimode=1080p60hz\0" \
+        "boot_usb=" \
+            "usb start; " \
+            "setenv devtype usb; " \
+            "for n in ${usb_list}; do "	\
+			"setenv devnum ${n}; " \
+			"setenv devno ${n}; " \
+			"load usb ${n} ${preloadaddr} boot.ini; " \
+			"source ${preloadaddr}; " \
+		"done\0" \
+        "boot_usb_distro=" \
+            "setenv devtype usb; " \
+            "for n in ${usb_list}; do " \
+			"setenv devnum ${n}; " \
+			"setenv devno ${n}; " \
+			"load usb ${n} ${preloadaddr} boot.scr; " \
+			"source ${preloadaddr}; " \
+		"done\0" \
+         "boot_spi=" \
+            "hdmitx edid; setenv bootargs ${initargs} console=tty0 logo=osd0,loaded,0x3d800000 " \
+                "osd_reverse=0 video_reverse=0; setenv bootargs ${bootargs} vout=${vout} " \
+                "hdmimode=${hdmimode} modeline=${modeline} voutmode=${voutmode}; osd open; " \
+                "osd clear; vout output ${outputmode}; run set_spi_params; sf probe; " \
+                "sf read ${preloadaddr} ${start_kernel} ${size_kernel}; sf read ${initrd_high} " \
+                "${start_initrd} ${size_initrd}; sf read ${dtb_mem_addr} ${start_dtb} ${size_dtb}; " \
+                "if test -e mmc 1:1 spiboot.img; then fdt addr ${dtb_mem_addr}; fdt resize; " \
+                "fdt set /emmc@ffe07000 status 'disabled'; fdt set /soc/cbus/spifc@14000 " \
+                "status 'okay'; fi; bootm ${preloadaddr} ${initrd_high} ${dtb_mem_addr};" \
+                "\0"\
+        "boot_order=" \
+            "boot_usb boot_usb_distro boot_mmc boot_mmc_distro boot_rawimage boot_pxe boot_spi" \
```

Comment:
As described above regarding boot order of MMC cards, we also like to keep the current boot order, but willing to offer the capability to change it by the user.

Reply:
You are free to change this PR at your discretion. For me, it is important to be able to easily activate any desired user order in the SPI (by simply running a text file that is easy to edit to fit your needs).

Reply:
Thanks again, actually I am doing a similar thing to offer a method to change the boot order in U-boot but not have spent much time for some other work. And I am planning to keep doing so in Petitboot by mid of June, but I am feeling that I would provide some way what you did soon. :)